keras/backend/mxnet_backend.py

```diff
@@ -3011,7 +3011,27 @@ def conv2d_transpose(x, kernel, output_shape, strides=(1, 1),
     # Raises
         ValueError: if `data_format` is neither `channels_last` or `channels_first`.
     """
-    raise NotImplementedError('MXNet Backend: conv2d_transpose operator is not supported yet.')
+    if data_format is None:
+        data_format = image_data_format()
+    _validate_data_format(data_format)
+
+    if padding not in {'same', 'valid'}:
+        raise ValueError('`padding` should be either `same` or `valid`.')
+
+    x = _preprocess_convnd_input(x, data_format)
```

Comment:
Please add detailed code comment taking an example input, kernel, strides, output shape for both channels first and channels last. Also mention kernel will always be in channels last format.

This will greatly help in code maintenance and debugging in future.